collect/cache/cuckoo.go

```diff
@@ -0,0 +1,97 @@
+package cache
+
+import (
+	"sync"
+
+	"github.com/honeycombio/refinery/metrics"
+	cuckoo "github.com/panmari/cuckoofilter"
+)
+
+// These are the names of metrics tracked for the cuckoo filter
+const (
+	CurrentLoadFactor = "cuckoo_current_load_factor"
+	FutureLoadFactor  = "cuckoo_future_load_factor"
+	CurrentCapacity   = "cuckoo_current_capacity"
+)
+
+// This wraps a cuckoo filter implementation in a way that lets us keep it running forever
+// without filling up.
+// A cuckoo filter can't be emptied (you can delete individual items if you know what they are,
+// but you can't get their names from the filter. Consequently, what we do is keep *two* filters,
```

Comment:
```suggestion
// but you can't get their names from the filter). Consequently, what we do is keep *two* filters,
```

Reply:
Thanks. Just so you don't think I'm being a jerk, I don't generally press the "commit suggestion" button because then I don't have the commit on my local branch except after it's been squashed, which means that I can't autodelete the branches.